components/onboarding-modal/OnboardingModal.js

```diff
@@ -241,7 +241,10 @@ class OnboardingModal extends React.Component {
     // https://github.com/shinnn/github-username-regex
     if (
       values.githubHandle !== '' &&
-      matches(values.githubHandle, /^[a-z\d](?:[a-z\d]|-(?=[a-z\d])){0,38}$/i) === false
+      matches(
```

Comment:
@Yetizone : Minor simplification, I think you can remove the `=== false` and negate the condition `!matches(...`